projects/vdk-heartbeat/src/vdk/internal/heartbeat/config.py

```diff
@@ -74,6 +74,19 @@ def __init__(self, config_file=None):
             self.get_value("RUN_TEST_TIMEOUT_SECONDS", "1200")
         )
 
+        # Ingestion configuration
```

Comment:
Currently, our only documentation for those configuration keys is in this file. Can you come with a bit more clear description in the comments for each new field?

 Imagine a new user using vdk-heartbeat for a first time would be using this.

Reply:
Sure